modules/monitoring-accessing-the-targeting-ui.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-targets.adoc
+
+[id="monitoring-accessing-the-targeting-ui_{context}"]
+= Accessing the Targeting UI in the Administrator perspective
```

Comment:
I'm wondering if instead of `Targeting UI`, we should call it `Monitoring Targets UI` or maybe `Metrics Targets UI`. We use "Targets" in the UI rather than "Targeting" and I'm thinking that including "Monitoring" / "Metrics" would make it very clear what type of targets we are discussing. WDYT?

Reply:
@kyoto That makes sense to me. I was trying to parallel previous content the refers to the "Alerting UI". I think mentioning metrics in there somewhere makes sense, so I'll try that.

Reply:
👍